src/service/viewport-impl.js

```diff
@@ -108,7 +108,7 @@ export class Viewport {
     this./*OK*/scrollLeft_ = null;
 
     /** @private {number} */
-    this.paddingTop_ = viewer.getPaddingTop();
+    this.paddingTop_ = viewer.getInitPaddingTop();
```

Comment:
Here: `this.paddingTop_ = Number(viewer.getParam('paddingTop') || 0)`. Should work, right?